source/javascripts/controllers/_ProvProfileController.js.erb

```diff
@@ -152,6 +152,10 @@ angular.module("BitriseWorkflowEditor").controller("ProvProfileController", func
 		return _.any(certificateService.certificates, function(aCertificate) {
 			return _.any(aCertificate.details, { serial: identitySerial });
 		});
+	}
+
+	viewModel.isIdentityExpired = function(identityExpiryDateTime) {
+		return new Date(identityExpiryDateTime) < new Date();
```

Comment:
`new Date(identityExpiryDateTime)` -> simply `identityExpiryDateTime`, since it is already a Date type object.